src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -632,12 +646,19 @@ private void addBackButton(ChestMenu menu, int slot, Player p, PlayerProfile pro
         if (isSlimefunRecipe) {
             SlimefunItem slimefunItem = SlimefunItem.getByItem(item);
 
-            if (slimefunItem == null) {
+            if (slimefunItem == null || slimefunItem.canUse(p, false)) {
```

Comment:
Think you meant to negate that second expression

Reply:
Wdym? That is meant to return the original item if the player has unlocked the item

Reply:
he means if slimefun item is null OR slimefunItem can NOT be used

right now it says if slimefun item is null or slimefunItem can be used

So in other words I think hes correct here

Reply:
My current if statement here means
if slimefunItem is null (it is not a slimefun item) OR slimefun item can be used (it is already unlocked)
return the original itemstack
else
return a barrier

Reply:
ah right then it does make sense; maybe add a small comment to indicate this because it can be confusing?

Reply:
The line right below it indicated everything?

Reply:
right my bad, its fine as is